### PR TITLE
Update year from copyright

### PR DESCRIPTION
**Pull request by lgcosta**
command help in this change:
`find . -type f -exec perl -i -pe's/Copyright 2005-201[0-7]/Copyright 2005-2018/' {} \;`

### CODE REVIEW

**Comment by sc979** [comment]
Please can you add a command to replace the word "developped" with "developed" too

**Comment by lgcosta** [comment]
@sc979 done!

File: www/class/centreon-knowledge/procedures_DB_Connector.class.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 MERETHIS
```
Comment (sc979) [inline www/class/centreon-knowledge/procedures_DB_Connector.class.php]: you missed one ;)

File: www/class/centreon-knowledge/procedures_Proxy.class.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
 /*
  * MERETHIS
  *
- * Source Copyright 2005-2010 MERETHIS
+ * Source Copyright 2005-2018 MERETHIS
```
Comment (sc979) [inline www/class/centreon-knowledge/procedures_Proxy.class.php]: That's the kind of licence for wich I need the advise of JMA and LPI...
Should we replace MERETHIS with CENTREON ? I don't know

File: www/include/configuration/configKnowledge/display-hostTemplates.php [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 MERETHIS
```
Comment (sc979) [inline www/include/configuration/configKnowledge/display-hostTemplates.php]: this one need an update ->  2005-2018 CENTREON

File: www/include/configuration/configKnowledge/display-serviceTemplates.php [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 MERETHIS
```
Comment (sc979) [inline www/include/configuration/configKnowledge/display-serviceTemplates.php]: same here

File: www/include/configuration/configKnowledge/display-services.php [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 MERETHIS
```
Comment (sc979) [inline www/include/configuration/configKnowledge/display-services.php]: same here

File: www/include/configuration/configKnowledge/popup.php [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 MERETHIS
```
Comment (sc979) [inline www/include/configuration/configKnowledge/popup.php]: same here

File: www/include/configuration/configKnowledge/proxy/proxy.php [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 MERETHIS
```
Comment (sc979) [inline www/include/configuration/configKnowledge/proxy/proxy.php]: same here

File: www/include/configuration/configKnowledge/popup_form.php [hunk 8]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 MERETHIS
```
Comment (sc979) [inline www/include/configuration/configKnowledge/popup_form.php]: same here

File: www/include/configuration/configObject/host_categories/DB-Func.php [hunk 9]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 Centreon
```
Comment (sc979) [inline www/include/configuration/configObject/host_categories/DB-Func.php]: same here

File: www/include/configuration/configObject/host_categories/formHostCategories.php [hunk 10]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 Centreon
```
Comment (sc979) [inline www/include/configuration/configObject/host_categories/formHostCategories.php]: same here

File: www/include/configuration/configObject/host_categories/hostCategories.php [hunk 11]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 Centreon
```
Comment (sc979) [inline www/include/configuration/configObject/host_categories/hostCategories.php]: same here

File: www/include/configuration/configObject/host_categories/listHostCategories.php [hunk 12]
```diff
@@ -1,7 +1,7 @@
 <?php
 /*
  * Copyright 2005-2009 Centreon
```
Comment (sc979) [inline www/include/configuration/configObject/host_categories/listHostCategories.php]: same here

File: www/class/centreon-knowledge/procedures_Proxy.class.php (hunk 2 above)
Comment (lgcosta) [reply www/class/centreon-knowledge/procedures_Proxy.class.php]: I believe we need create a new line with a range of years, example:
```
Source Copyright 2005-2010 MERETHIS
Source Copyright 2011-2018 CENTREON
```

**Comment by lgcosta** [comment]
thanks @sc979. I made the changes with year 200[6-9]

**Comment by cgagnaire** [comment]
We close this one as it is too old now.
We keep the command line anyway :)
Thanks!

**Pull request closed without merging**